JSDemos/Demos/List/ListSelection/description.md

```diff
@@ -1 +1,9 @@
-The **List** UI component offers three item selection modes: _&laquo;all&raquo;_, _&laquo;single&raquo;_ and _&laquo;multi&raquo;_. Note that when the current mode is _&laquo;all&raquo;_, the **List** allows a&nbsp;user to&nbsp;select all items at&nbsp;once by&nbsp;clicking the _Select All_ check box. The **selectAllMode** property allows you to&nbsp;decide whether all items should be&nbsp;selected across all pages or&nbsp;within the currently rendered pages only.
\ No newline at end of file
+The **List** offers three selection modes: *all*, *single* and *multi*. The UI of this demo consists of the **List**, Selected IDs, and Options. To see selection modes in action, choose one from the dropdown menu named Selection Mode. You can then select a list item or multiple items from the **List**. You can see the IDs of selected items in the Selected IDs section. This done by a function that handles the [onSelectionChanged](/Documentation/ApiReference/UI_Widgets/dxList/Configuration/#onSelectionChanged) event.
+
+Check a checkbox a radio button next to a list item to select it. This is possible because the [showSelectionControls](/Documentation/ApiReference/UI_Widgets/dxList/Configuration/#showSelectionControls) property is set to **true**. Set the property to **false** to disable selection controls. In this case, you can click on items you want to select. Selected items are grayed out. 
+
+When you choose a selection mode from the dropdown menu, the value you select is assigned to the [selectionMode](/Documentation/ApiReference/UI_Widgets/dxList/Configuration/#selectionMode) property. When you choose *all*, "Select all" checkbox is rendered at the top of the **List**. This value only works when **showSelectionControls** is set to **true**. In this mode, the "Select All Mode" dropdown menu becomes active. When you choose the *page* option, this value is assigned to the [selectAllMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#selectAllMode) property. This allows you to select all items only on the currently rendered page. To select all items on all available pages, set this property to *allPAges*. 
+
+Select the *multiple* mode to be able to check several checkboxes at the same time. 
+
+Or use the *single* mode if you need to select only one item at a time. In this case, the selection is controlled with radio buttons.
```

Comment:
You describe too much of the demo. Let's focus on the capabilities. I think you can take the [Selection - User Interaction](https://js.devexpress.com/Documentation/Guide/UI_Components/List/Selection/#User_Interaction) article, adapt it a bit to satisfy the guidelines, add information about `selectedItemKeys` and `onSelectionChanged` to the end, and you will have a decent description.